docs/_modules/record-validation/con-example-kafkaprotocolfilter-resource.adoc

```diff
@@ -0,0 +1,39 @@
+// file included in the following:
+//
+// assembly-configuring-record-validation-filter.adoc
+
+[id='con-example-kafkaprotocolfilter-resource-{context}']
+= Example `KafkaProtocolFilter` resource
+
+If your instance of Kroxylicious runs on Kubernetes, you must use a `KafkaProcotolFilter` resource to contain the filter configuration.
+
+Here's a complete example of a `KafkaProtocolFilter` resource configured for record encryption with Vault KMS:
+
+.Example `KafkaProtocolFilter` resource
```

Comment:
```suggestion
.Example `KafkaProtocolFilter` resource for record validation
```

Reply:
Is is really an example if it's got placeholders though? Maybe we should describe it as an outline or something